packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -218,4 +198,25 @@ export default {
 	min-width: 10rem;
 	margin-top: var(--unit-8);
 }
+
+// block modifier
+.kt-button.kt-button--is-block {
```

Comment:
Can you use `&.&--is-block` and put it back to its original location? AFAIK this should work and is more in-line with the rest of SCSS in this repo.

Reply:
I don;t think there is any benefit to that. It render the same code and I don't find it any more readable or understandable.